test/integration/clustermanager_test.go

```diff
@@ -491,6 +492,31 @@ var _ = ginkgo.Describe("ClusterManager Detached Mode", func() {
 	ginkgo.BeforeEach(func() {
 		var ctx context.Context
 		ctx, cancel = context.WithCancel(context.Background())
+
```

Comment:
I think we should have a separated file to test detached mode. But it could be a followup

Reply:
Sure, I have split this into two files now.